values.md

```diff
@@ -1,27 +1,26 @@
 # Kubernetes Community Values
 
-Kubernetes Community culture is frequently cited as a substantial contributor to the meteoric rise of this Open Source project.  Below are the distilled values which have evolved over the last many years in our community pushing our project and peers toward constant improvement.
+The culture of Kubernetes contributes substantially to the project's success.  Below are the distilled values which have evolved over the years in our community, pushing our project and peers toward constant improvement.
```

Comment:
I believe it is important to retain the reference to Kubernetes _Community_ with regards to culture. The second part could be put into active voice: The values below are distilled from years of community evolution and drive constant improvement in the project.

Reply:
"are distilled from" still uses passive voice. Maybe something like:
```suggestion
Kubernetes Community culture contributes substantially to the project's success. The following values evolved over years of community evolution, pushing our project and peers toward constant improvement.
```